deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   annotations:
     deis.io/objectstorage: "{{.storage}}"
 type: Opaque
-data: {{ if eq .storage "gcs" }}
+data: {{ if eq .storage "minio" }}
+  access-key-id: OFRaUlkySlJXTVBUNlVNWFI2STUK
```

Comment:
@kmala is it possible to use the same generator to generate both keys? not necessary for this PR IMO, but it would be nice to have that